src/drivers/intel/ssp/mn.c

```diff
@@ -340,6 +340,22 @@ static int find_bclk_source(uint32_t bclk,
 	return -1;
 }
 
+/**
+ * \brief Finds index of SSP clock with the given clock source encoded index.
+ * \return the index in ssp_freq if could find it, -1 otherwise.
+ */
+static int find_clk_ssp_index(uint32_t src_enc)
+{
+	int i;
+
+	/* searching the smallest possible bclk source */
```

Comment:
is this a copy/paste error ? we're not really searching for the smallest here?

Reply:
> is this a copy/paste error ? we're not really searching for the smallest here?

yep, let me correct it.